WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Service/BlogDashboardService.swift

```diff
@@ -97,27 +97,83 @@ private extension BlogDashboardService {
 
     func parse(_ entity: BlogDashboardRemoteEntity?, blog: Blog, dotComID: Int) -> [DashboardCardModel] {
         let personalizationService = BlogDashboardPersonalizationService(repository: repository, siteID: dotComID)
-        var cards: [DashboardCardModel] = DashboardCard.allCases.compactMap { card in
-            guard personalizationService.isEnabled(card) else {
+
+        // Map `DashboardCard` instances to `DashboardCardModel`
+        var allCards: [DashboardCardModel] = DashboardCard.allCases.compactMap { card -> DashboardCardModel? in
+            guard card != .dynamic else {
                 return nil
             }
+            return self.dashboardCardModel(
+                from: card,
+                entity: entity,
+                blog: blog,
+                dotComID: dotComID,
+                personalizationService: personalizationService
+            )
+        }
 
-            guard card.shouldShow(
-                for: blog,
-                apiResponse: entity,
-                isJetpack: isJetpack,
-                isDotComAvailable: isDotComAvailable,
-                shouldShowJetpackFeatures: shouldShowJetpackFeatures
-            ) else {
-                return nil
+        // Maps dynamic cards to `DashboardCardModel`.k
```

Comment:
`.k` is a typo?

Reply:
Yes sir!

Reply:
@alpavanoglu This was resolved in this [PR](https://github.com/wordpress-mobile/WordPress-iOS/pull/22229)